protos/kuksa.proto

```diff
@@ -0,0 +1,38 @@
+// Copyright 2015 gRPC authors.
```

Comment:
As this is _your_protofiole (and not written 2015 by some grpc guys), the correct license for this would be EDL. (That makes it easy to use the protofile for others, even with more permissive or strict licenses than EPL)

# Copyright Robert Bosch GmbH, 2021. Part of the Eclipse Kuksa Project.
#
# All rights reserved. This configuration file is provided to you under the
# terms and conditions of the Eclipse Distribution License v1.0 which
# accompanies this distribution, and is available at
# http://www.eclipse.org/org/documents/edl-v10.php